File: app/src/main/java/intellij/class11/exceptions/exercise3/java/Main.java
package intellij.class11.exceptions.exercise3.java;

public class Main {
    public static void main(String[] args) {
        CalculoMatematico calculoMatematico = new CalculoMatematico();
        //Teste 1
        calculoMatematico.divisao(4,0);
        //Teste 2
        try {
            calculoMatematico.divisao2(4,0);
        } catch (ArithmeticException exception) {
            System.out.println(exception.getMessage());
        }
    }
}

File: app/src/main/java/intellij/class11/exceptions/exercise3/kotlin/Main.kt
package class11.exceptions.exercise3.kotlin

import intellij.class11.exceptions.exercise3.kotlin.CalculoMatematico

fun main(args: Array<String>) {
    //Teste 1
    CalculoMatematico().divisao(4, 0)
    //Teste 2
    try {
        CalculoMatematico().divisao2(4, 0)
    } catch (exception: ArithmeticException) {
        println(exception.message)
    }
}